### PR TITLE
Add shortID in VPCNetworkConfiguration CR

### PR DESCRIPTION
**Pull request by lxiaopei**
Add shortID in VPCNetworkConfiguration CR for user to input short log identifer they want to use to isolate VPC logs.

### CODE REVIEW

File: build/yaml/crd/nsx.vmware.com_vpcnetworkconfigurations.yaml [hunk 1]
```diff
@@ -89,6 +89,11 @@ spec:
                 maxItems: 5
                 minItems: 0
                 type: array
+              shortID:
```
Comment (jianjuns) [inline build/yaml/crd/nsx.vmware.com_vpcnetworkconfigurations.yaml:92]: We should use a more explicit name like `logID`. Could you check how this is called in NSX-T?

Comment (lxiaopei) [reply build/yaml/crd/nsx.vmware.com_vpcnetworkconfigurations.yaml:92]: https://gitlab.eng.vmware.com/core-build/nsx/-/blob/nsx-keeper/mp/policy/policy-api/src/main/resources/api_spec/root/nsx-policy/policy-vpc.yml#L123, yea, I checked in NSX-T it is shortID, so use the same name.

**Review by seanpang-vmware — APPROVED** [review]

File: build/yaml/crd/nsx.vmware.com_vpcnetworkconfigurations.yaml [hunk 2]
```diff
@@ -89,6 +89,11 @@ spec:
...
+                description: ShortID specifies Identifier to use when displaying vpc
```
Comment (jianjuns) [inline build/yaml/crd/nsx.vmware.com_vpcnetworkconfigurations.yaml:93]: vpc -> VPC

**Review by jianjuns — APPROVED** [review]
A nit.

File: build/yaml/crd/nsx.vmware.com_vpcnetworkconfigurations.yaml (hunk 2 above)
Comment (lxiaopei) [reply build/yaml/crd/nsx.vmware.com_vpcnetworkconfigurations.yaml:93]: Thanks，changed.

**Review by seanpang-vmware — APPROVED** [review]

**Review by jianjuns — APPROVED** [review]

**Pull request merged**